package.json

```diff
@@ -16,6 +17,7 @@
     "@types/react-router-dom": "4.3.1",
     "@types/redux-logger": "3.0.6",
     "bootstrap": "4.2.1",
+    "classnames": "2.2.6",
```

Comment:
I was wondering how long it would be until someone added `classnames`. ;-)

Reply:
😄